src/yadism/coefficient_functions/asy/f2_nc.py

```diff
@@ -7,6 +7,8 @@
 
 
 class AsyLLGluon(pc.NeutralCurrentBaseAsy):
+    hs3 = adani.HighScaleSplitLogs(3, '2', 'g', "abmp")
```

Comment:
not sure if here it's nicer to have this the top level, 
creating an additional class and avoid repetition 

```
class AsyGluon(pc.NeutralCurrentBaseAsy):
        hs3 = adani.HighScaleSplitLogs(3, '2', 'g', "abmp")
...
class AsyLLGluon(AsyGluon):

....
```

Reply:
Ok done!